code/game/objects/items/broom.dm

```diff
@@ -68,13 +68,17 @@
  * * user - The user of the pushbroom
  * * A - The atom which is located at the location to push atoms from
  */
-/obj/item/pushbroom/proc/sweep(mob/user, atom/A)
+/obj/item/pushbroom/proc/sweep(mob/user, atom/atom)
 	SIGNAL_HANDLER
 
-	var/turf/current_item_loc = isturf(A) ? A : A.loc
+	do_sweep(src, user, atom, user.dir)
+
+/// Sweep objects in the direction we're facing towards our direction
+/proc/do_sweep(obj/broomer, mob/user, atom/target, sweep_dir)
```

Comment:
Should include what the args are in the autodoc like typically done

![image](https://github.com/tgstation/tgstation/assets/51863163/e3b003e4-860c-4fa7-a705-f827f1ff7df0)